lib/grid_search.py

```diff
@@ -48,12 +48,22 @@ def train(self):
         best_error = numpy.inf
         best_params = dict()
         train, test = self.recommender.naive_split()
+        predictions = None
+        all_results = [['n_factors', '_lambda', 'rmse', 'train_recall', 'test_recall', 'recall_at_200', 'ratio',
```

Comment:
I would prefer if this is stored as a method static variable for the method **get_evaluation_report**, in order for it to remain consistent in case you changed the arguments of the evaluation_report later on.